users/models.py

```diff
@@ -8,3 +10,34 @@ class Person(models.Model):
 
     class Meta:
         abstract = True
+
+
+class Club(models.Model):
+    name = models.CharField(_('name'),
+                            max_length=128)
+    contact = models.OneToOneField('MainCoach',
```

Comment:
Nee, dat lijkt me niet handig :') 
Omdat we toch geen aparte `MainCoach`  meer hebben, is het dan voldoende om voor de `Coach` class een FK naar `Club` te hebben?